ytmusicapi/parsers/playlists.py

```diff
@@ -91,7 +97,9 @@ def parse_playlist_items(results, menu_entries: Optional[List[List]] = None):
             "isExplicit": isExplicit,
             "videoType": videoType,
             "views": views,
+            "track_position": track_position,
```

Comment:
Not sure about the best key to use here. `track_position`, `track_index`, `track_number`?

Spotify uses `track_number`: https://developer.spotify.com/documentation/web-api/reference/get-an-album

Not sure if we have any other similar examples in `ytmusicapi` - should avoid inconsistencies. AFAIK the index is currently not returned on `get_playlist` either

What do you think?

Reply:
I don't feel strongly pulled in any direction. I was fairly indecisive when originally picking the key before settling on track_position. Personally, I've never liked track_number from Spotify's API, but there's definitely an argument to be made for it being less ambiguous here. 

In regards to playlist return, I figured it would just return something similar for playlists, but looking at it now, I don't think the response includes indexing for actual playlists (at least from what's visible on the UI). It might be worth adding an is_album param to parse_playlist_items to avoid a nested track_position lookup that'll never return anything. Thoughts?

Reply:
apple also uses `trackNumber`: https://developer.apple.com/documentation/applemusicapi/songs/attributes

let's not be the odd ones out here

Reply:
and you're right, sadly playlists don't include `trackNumber` - so it makes sense to do as you did